Dockerfile

```diff
@@ -12,10 +17,36 @@ ENV MODEL_NAME=${MODEL_NAME}
 ARG MODEL_TAG
 ENV MODEL_TAG=${MODEL_TAG}
 
+# Update and upgrade the system packages (Worker Template)
+RUN apt-get update -y && \
```

Comment:
revisit to strip out things not needed for a serverless worker, things only needed for other installs